src/main.js

```diff
@@ -65,15 +66,26 @@ let load = loader({
       aws:           cfg.aws,
     }),
   },
+
+  pulseClient: {
+    requires: ['cfg', 'monitor'],
+    setup: ({cfg, monitor}) => {
+      return new Client({
+        namespace: cfg.pulse.namespace,
+        monitor,
+        credentials: pulseCredentials(cfg.pulse),
```

Comment:
Not relevant to this system, but in lib-pulse, it might be a good idea to allow passing in objects in the shape of cfg.pulse and handle the mapping to a connection string internally.

Reply:
That's what this accomplishes.  There are a few ways to provide credentials, and this is only one of them, so the *Credentials methods map them all to the same format (a callable).  This made more sense when fetching dynamic credentials from tc-pulse, but I think the model is natural enough to keep even without that feature.

Reply:
(It's based on the model used in taskcluster-lib-azure)